backend/Perflow/Perflow.csproj

```diff
@@ -26,10 +26,10 @@
     <Folder Include="Infrastructure\" />
   </ItemGroup>
 
-  <ItemGroup>
-    <Reference Include="Shared">
-      <HintPath>..\Shared\Shared\bin\Debug\net5.0\Shared.dll</HintPath>
-    </Reference>
+  <ItemGroup>
+    <Reference Include="Shared">
+      <HintPath>..\Shared\Shared\bin\Debug\net5.0\Shared.dll</HintPath>
```

Comment:
I'm not sure it will work for production, since 'Debug' folder is hard coded

Reply:
Yes, I'm not sure, too. Can you advise me something, please? I thought this reference would be overridden later before production?

Reply:
All done. Project reference has been changed to 
    <ItemGroup>
          <ProjectReference Include="..\Shared\Shared\Shared.csproj" />
    </ItemGroup>